src/components/pop-up/index.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled'
+import React from 'react'
+
+type Props = {
+  children: React.ReactNode
+  placeInView: [
+    isDisplayed: boolean,
+    setDisplay: React.Dispatch<React.SetStateAction<boolean>>,
+  ]
```

Comment:
this will cause magic placements
`placeInView={["what am I?", () => {now I am a function}, values()]}`
it's better to break them down to 2 props rather than try to make the return value from `useState` a match

Reply:
hmmm i can make it an object. this way there shouldnt be any magic

Reply:
and object is better, but why not just 2 props?

Reply:
i see no difference at this point. in both cases we have types as reference

Reply:
I would suggest two props as well